backend/app/Http/Controllers/Api/ProceduresApiController.php

```
@@ -0,0 +1,20 @@
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\Models\Procedure;
+
+class ProceduresApiController extends Controller
+{
+
+    public function getProcedures()
+    {
+        return Procedure::paginate();
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\Procedure' in method 'getProcedures'. (Clean Code Rules)

(StaticAccess)